charts/media-api-service/values.yaml.Example

```diff
@@ -0,0 +1,46 @@
+namespace: "hanwha"
+# fullnameOverride: imagename
+
+image:
+  replicaCount: 1
+  repository: ghcr.io/htaic/media-api-service
+  pullPolicy: Always
+  containerPort: 3000
+  tag: "latest"
+  domainPath: ""
```

Comment:
Are you planning to expose this service or is it internal only... if its internal, remove domain path and the ingress route and middleware.

Also please confirm you are in fact using port 3000

Reply:
Yep, that's NestJs' default port.

Reply:
The latest commit had the domainPath updated to "media-api".